ansys/mapdl/core/mesh_grpc.py

```diff
@@ -137,6 +137,30 @@ def nnum_all(self) -> np.ndarray:
 
         return nnum
 
+    @property
+    def enum_all(self) -> np.ndarray:
+        """Array of all element numbers, even those not selected.
+
+        Examples
+        --------
+        >>> mapdl.mesh.enum_all
+        array([    1,     2,     3, ..., 19998, 19999, 20000])
+        """
+        self._ignore_cache_reset = True
+        self._mapdl.cm("__ELEM__", "ELEM", mute=True)
```

Comment:
Part of the confusion here is I run `nsel` instead of `esel`. Really, we need to make sure we have all elements selected before returning the list of element numbers.  I'm fixing this.